into/backends/tests/test_sql.py

```diff
@@ -185,3 +188,30 @@ def test_resource_on_dialects():
             resource.dispatch('mysql+pymysql://foo'))
     assert (resource.dispatch('never-before-seen-sql://foo') is
             resource.dispatch('mysql://foo'))
+
+
+@pytest.yield_fixture
+def sqlite_file():
+    try:
+        yield 'sqlite:///db.db'
+    finally:
+        os.remove('db.db')
+
+
+def test_append_from_select(sqlite_file):
+    # we can't test in memory here because that creates two independent
+    # databases
+    raw = np.array([(200.0, 'Glenn'),
+                    (314.14, 'Hope'),
+                    (235.43, 'Bob')], dtype=[('amount', 'float64'),
+                                             ('name', 'S5')])
+    raw2 = np.array([(800.0, 'Joe'),
+                     (914.14, 'Alice'),
+                     (1235.43, 'Ratso')], dtype=[('amount', 'float64'),
+                                                 ('name', 'S5')])
+    t = into('%s::t' % sqlite_file, raw)
+    s = into('%s::s' % sqlite_file, raw2)
+    t = append(t, s.select())
```

Comment:
Hrm, now that I think about this I bet we could make this work in either case by dispatching on `sqlalchemy.Selectable` rather than `sqlalchemy.Select`.